SpatialGDK/Source/SpatialGDK/Public/SpatialView/ConnectionHandlers/SpatialOSConnectionHandler.h

```diff
@@ -0,0 +1,190 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "SpatialView/ConnectionHandlers/AbstractConnectionHandler.h"
+#include "SpatialView/OpList/OpList.h"
+#include "SpatialView/OpList/WorkerConnectionOpList.h"
+
+namespace SpatialGDK
+{
+
+class SpatialOSConnectionHandler : public AbstractConnectionHandler
+{
+public:
+	explicit SpatialOSConnectionHandler(Worker_Connection* Connection)
+	: Connection(Connection), WorkerId(UTF8_TO_TCHAR(Worker_Connection_GetWorkerId(Connection)))
+	{
+		const Worker_WorkerAttributes* Attributes = Worker_Connection_GetWorkerAttributes(Connection);
+		for (uint32 i = 0; i < Attributes->attribute_count; ++i)
+		{
+			WorkerAttributes.Push(FString(UTF8_TO_TCHAR(Attributes->attributes[i])));
+		}
+	}
+
+	void Advance() override
```

Comment:
Is it convention to add virtual?

Reply:
Oh yeah I keep forgetting about our weird style guides. Will go back and do that.